### PR TITLE
Jarman - Ch.02 Practice

### PR DESCRIPTION
**Pull request by Jarmander333**

### CODE REVIEW

**Comment by Jarmander333** [comment]
ave = x+y+z/3 -- Should be '(x+y+z)/3', not "OK" overlooked it.

**Comment by Jarmander333** [comment]
ave = x+y+z/3 -- Should be 'ave = (x+y+z)/3', not "OK".

File: 2.0_Jedi_Training.py [hunk 1]
```diff
@@ -26,22 +29,21 @@
...
+type(a)       = <class 'float'>
 
 
 4.) What is the mistake in the following code. Fix it!
 
-x,y = (4,5)
-a = 3(x + y)
-a
+x,y = (4,5) -- OK
+a = 3(x + y) -- Should be 'a = 3*x + 3*y'
```
Comment (hermonm) [inline 2.0_Jedi_Training.py:39]: a=3*(x+y)

File: 2.0_Jedi_Training.py [hunk 2]
```diff
@@ -26,22 +29,21 @@
...
 
 
 5.) What is the mistake in the following code so it will calculate the average. Fix it!
 
-x,y,z =(3,4,5)
-ave = x+y+z/3
-ave
-
+x,y,z =(3,4,5) -- OK
+ave = x+y+z/3 -- OK
```
Comment (hermonm) [inline 2.0_Jedi_Training.py:46]: This average equation is incorrect.

**Comment by hermonm** [comment]
need to fix stuff and upload your ch 2 log

**Pull request closed without merging**